src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -716,3 +751,15 @@ def self_test(self):
 
         c = orchestrator.TrivialReadCompletion(result=True)
         assert c.has_result
+
+    @orchestrator._cli_write_command(
+        'upgrade check',
+        'name=image,type=CephString,req=false',
+        desc='Check service versions vs available and target containers')
+    def _upgrade_check(self, image=None):
+        if image and re.search('^\d+\.\d+\.\d+$', image):
+            image = self.container_image_base + ':v' + image
```

Comment:
1. Not every registry adds a `v` prefix in the tag.
2. the logic seems strange to me. This only works, if you pass a version number as image, like 
    ```
    ceph orchestrator upgrade check --image=14.2.4
    ```
    this looks odd to me. Can we find a more user-friendly CLI here?

Reply:
i can make a separate arg for version?